src/schemas/json/tslint.json

```
@@ -3444,6 +3478,23 @@
             }
           ]
         },
+        "no-async-without-await": {
+          "description": "Functions marked async must contain an await or return statement.",
```

```suggestion
          "description": "Force functions marked async contain an await or return statement\nhttps://palantir.github.io/tslint/rules/no-async-without-await/",
```

The same issue as above.